RubeesEat/Model/DB/DbPersonRepository.cs

```diff
@@ -1,11 +1,25 @@
+using System.Data;
+using System.Security.Claims;
+
 namespace RubeesEat.Model.DB;
 
-public class DbPersonRepository(IDbConnectionFactory connectionFactory) : IPersonRepository
+public class DbPersonRepository: IPersonRepository
 {
+    private readonly IHttpContextAccessor? _httpContextAccessor;
+    private readonly IDbConnectionFactory _connectionFactory;
+    public DbPersonRepository(IHttpContextAccessor httpContextAccessor, IDbConnectionFactory connectionFactory)
+    {
+        _httpContextAccessor = httpContextAccessor;
+        _connectionFactory = connectionFactory;
+    }
+    public DbPersonRepository(IDbConnectionFactory connectionFactory)
+    {
+        _connectionFactory = connectionFactory;
+    }
     public IReadOnlyList<Person> GetAll()
     {
```

Comment:
There should be empty lines between the method declarations and fields/method declaration.